### DISCUSSION TITLE
Add a bibtex reference for citing this software

### DISCUSSION THREAD

**Discussion by Jurian** [optimatika/ojAlgo#504] [General, ojAlgo and related code]
I am currently writing my PhD dissertation where I make use of ojalgo to solve ILP problems. I could not find a citation in the readme or the official website.

Maybe this can be added? I would like to give proper credit.

Cheers!

**Comment by apete** [comment]
Don't really know how bibtex works. Do I need to register ojAlgo somewhere? Googled the syntax and came up with this:
```
@software{ojAlgo,
  author = {{Optimatika}},
  title = {{ojAlgo} oj! Algorithms},
  url = {https://www.ojalgo.org},
  year = {2003--2023}
}
```
or
```
@software{ojAlgo,
  author = {{Optimatika}},
  title = {{ojAlgo} oj! Algorithms},
  url = {https://www.ojalgo.org},
  version = {52.0.1},
  date = {2022-10-20}
}
```
The release date of other specific versions can be found in the changelog.
